app/views/home/maintainers.html.erb

```diff
@@ -21,93 +21,93 @@
     </p>
   </div>
   <div class="mb-6 col-md-6 float-md-left">
     <%= image_tag("promote-work.svg", class: "img-responsive mb-3", height: "100px") %>
-      <h3 class="alt-h3 mb-2">Promote your work</h3>
+    <h3 class="alt-h3 mb-2"><%=t "Promote your work" %></h3>
       <p class="text-gray">
-      Consider creating a Twitter account, blog posts and website to <a href="https://opensource.guide/finding-users/#help-people-find-and-follow-your-project">help people find and follow your project</a>.
+      <%=t "Consider creating a Twitter account, blog posts and website to" %> <a href="https://opensource.guide/finding-users/#help-people-find-and-follow-your-project"><%=t "help people find and follow your project" %></a>
     </p>
   </div>
 </div>
 
 <div class="py-6 px-3 text-center border-top border-bottom bg-gray-light">
   <div class="container-lg mx-auto gut-lg py-md-6 text-center">
-    <h2 class="alt-h2">Grow your community</h2>
+    <h2 class="alt-h2"><%=t "Grow your community" %></h2>
     <p class="alt-lead text-gray col-md-8 mx-auto mb-6">
-      Everybody was new once. Ease someone else's first-time contribution jitters.
+      <%=t "Everybody was new once" %>
     </p>
     <div class="mb-6 col-md-6 float-md-left">
       <%= image_tag("document-everything.svg", class: "img-responsive mb-3", height: "100px") %>
-      <h3 class="alt-h3 mb-2">Document everything</h3>
+      <h3 class="alt-h3 mb-2"><%=t "Document everything" %></h3>
       <p class="text-gray">
-        <a href="http://opensourcesurvey.org/2017/">GitHub's 2017 Open Source Survey</a> showed incomplete or confusing documentation is the biggest problem for open source users. Ensure you are <a href="https://opensource.guide/best-practices/#documenting-your-processes">documenting your processes</a> for how to contribute to your project.
+      <a href="http://opensourcesurvey.org/2017/"><%=t "Open Source Survey" %></a><%=t "showed_incomplete_or" %> <%=t "Ensure_you_are_a" %> <a href="https://opensource.guide/best-practices/#documenting-your-processes"><%=t "Ensure_you_are_b" %></a> <%=t "Ensure_you_are_c" %>
       </p>
     </div>
     <div class="mb-6 col-md-6 float-md-left">
       <%= image_tag("get-help.svg", class: "img-responsive mb-3", height: "100px") %>
-      <h3 class="alt-h3 mb-2">Get help</h3>
+      <h3 class="alt-h3 mb-2"><%=t "Get help" %></h3>
       <p class="text-gray">
-        Look out for promising potential contributors or maintainers to <a href="https://opensource.guide/building-community/#share-ownership-of-your-project">share ownership of your project</a>. Make an effort to welcome first-time contributors in particular.
+        <%=t "Look out for promising potential" %> <a href="https://opensource.guide/building-community/#share-ownership-of-your-project"><%=t "share ownership of your project" %></a> <%=t "Make an effort to welcome" %>
```

Comment:
We can handle a text with link  better using foo_boo_html key.  Rails handles "_html" key's value as html. There are many place to improve.